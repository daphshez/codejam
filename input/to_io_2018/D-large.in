100
1
2
5
100
86
7
45
34
89
49
91
35
3
76
79
75
53
70
95
14
72
69
8
10
36
18
28
16
73
63
33
44
55
32
98
50
65
97
21
30
78
59
22
60
74
19
88
52
66
77
29
26
82
42
12
90
61
23
48
62
56
83
51
58
15
46
67
87
13
25
4
20
43
41
68
93
27
38
84
6
24
39
47
80
85
99
37
40
17
54
92
57
31
64
71
81
11
9
94
96
