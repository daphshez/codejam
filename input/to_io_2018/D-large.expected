Case #1: 1.0
Case #2: 3.3999999999999995
Case #3: 9.842023690595118
Case #4: 212.25369321351366
Case #5: 182.42891173496534
Case #6: 14.112811216315748
Case #7: 95.08403753636587
Case #8: 71.64939250084898
Case #9: 188.81994276010283
Case #10: 103.60560580065174
Case #11: 193.0806279707403
Case #12: 73.77984305194842
Case #13: 5.553571428571428
Case #14: 161.12544165730057
Case #15: 167.51648874867112
Case #16: 158.99509130072073
Case #17: 112.12713287088657
Case #18: 148.34332828683984
Case #19: 201.60199374553392
Case #20: 29.036499164586342
Case #21: 152.60403586397456
Case #22: 146.21297320885398
Case #23: 16.24608055677883
Case #24: 20.51073478188787
Case #25: 75.9102865207578
Case #26: 37.560238812833965
Case #27: 58.86649745011762
Case #28: 33.29852493599431
Case #29: 154.73438843411
Case #30: 133.43082209613263
Case #31: 69.5189342169076
Case #32: 92.95363770684803
Case #33: 116.38788378448983
Case #34: 67.3884674673833
Case #35: 207.99301438564936
Case #36: 105.73599104130007
Case #37: 137.69154337014206
Case #38: 205.86267450197855
Case #39: 43.95242293724571
Case #40: 63.12750514655364
Case #41: 165.38614035566076
Case #42: 124.90936467210327
Case #43: 46.08307642004463
Case #44: 127.03973104346016
Case #45: 156.86474023606928
Case #46: 39.691010125827525
Case #47: 186.68959952759224
Case #48: 109.99675444195285
Case #49: 139.8219023458945
Case #50: 163.25579133351948
Case #51: 60.997007563986884
Case #52: 54.6054337683927
Case #53: 173.9075303840542
Case #54: 88.69282716625956
Case #55: 24.774000891543913
Case #56: 190.9502855698219
Case #57: 129.17009602695055
Case #58: 48.21370084801715
Case #59: 101.47521800169334
Case #60: 131.30045969010234
Case #61: 118.51825648275957
Case #62: 176.0378764860856
Case #63: 107.86637387514271
Case #64: 122.77899684066732
Case #65: 31.16755901502421
Case #66: 97.21443405646536
Case #67: 141.95226028060483
Case #68: 184.55925585765314
Case #69: 26.905323111826092
Case #70: 52.47487667648283
Case #71: 7.701883116883117
Case #72: 41.821735966770326
Case #73: 90.823234335174
Case #74: 86.56241591992568
Case #75: 144.08261722043414
Case #76: 197.341311599119
Case #77: 56.73597339487219
Case #78: 80.1711544682362
Case #79: 178.16822206639964
Case #80: 11.978408755110435
Case #81: 50.344299904758785
Case #82: 82.30157992933864
Case #83: 99.34482747993226
Case #84: 169.64683653624462
Case #85: 180.29856714348463
Case #86: 210.12335395298686
Case #87: 78.0407234868488
Case #88: 84.43200028773533
Case #89: 35.42941384172361
Case #90: 114.25750928147197
Case #91: 195.21096997623837
Case #92: 120.64862747184169
Case #93: 65.25799142394399
Case #94: 135.56118330428595
Case #95: 150.47368249304742
Case #96: 171.77718374090043
Case #97: 22.64249112622153
Case #98: 18.378643995687792
Case #99: 199.47165285163814
Case #100: 203.73233429205342
